koku/api/currency/models.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2021 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Models for AWS cost entry tables."""
+# from uuid import uuid4
+from django.db import models
+from django.db.models import JSONField
+
+
+class ExchangeRates(models.Model):
+    AUD = "aud"
+    CAD = "cad"
+    CHF = "chf"
+    CNY = "cny"
+    DKK = "dkk"
+    EUR = "eur"
+    GBP = "gbp"
+    HKD = "hkd"
+    JPY = "jpy"
+    NOK = "nok"
+    NZD = "nzd"
+    SEK = "sek"
+    SGD = "sgd"
+    USD = "usd"
+    ZAR = "zar"
+    SUPPORTED_CURRENCIES = (
```

Comment:
Maybe you can use https://github.com/project-koku/koku/blob/main/koku/api/currency/currencies.py#L8 and map them into tuples of tuples?

Something like this.

```
SUPPORTED_CURRENCIES = tuple([(curr.code.toLower(), curr.code)for curr in CURRENCIES])
```